src/simulator/AbstractGUI.java

```diff
@@ -8,6 +8,8 @@
 
 import java.awt.event.*;
 
+//TODO: there is a scrollpane (bool) and scrollPane (JScrollPane)
```

Comment:
Generally a TODO is a note that there is work to be done still but this is good enough...and you record your thoughts for later.  This looks more like a note...